Packs/CommonTypes/IncidentFields/incidentfield-Campaign_Name.json

```diff
@@ -0,0 +1,27 @@
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "campaignname",
+    "closeForm": false,
+    "content": true,
+    "editForm": true,
+    "group": 0,
+    "hidden": false,
+    "id": "incident_campaignname",
+    "isReadOnly": false,
+    "locked": false,
+    "name": "Campaign Name",
+    "neverSetAsRequired": false,
+    "openEnded": false,
+    "ownerOnly": false,
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "markdown",
+    "unmapped": false,
+    "unsearchable": true,
```

Comment:
Maybe should also be searchable. Not sure if MD fields can be searchable though. If not, it may be problematic, because this field is a very generic field and it might prevent users from using it, and they will have to resort to creatiing a new field

Reply:
No need